.github/workflows/ecs.yml

```
@@ -0,0 +1,50 @@
+name: Deploy to ECS
+
+on: 
+  workflow_call:
+    inputs:
+      iam_role:
+        required: true
+        type: string
+      region:
+        required: false
+        type: string
+        default: us-east-1
+      ecs-cluster:
+        required: true
+        type: string
+      files:
+        required: false
+        type: string
+        default: docker-compose.yml:docker-compose.ecs.yml
+      env:
+        required: false
+        type: string
+        default: ''
+env:
+  GIT_SHA: ${{ github.sha }}
+
+permissions:
+      id-token: write
+      contents: read
```

Indentation needs fixing

```suggestion
permissions:
  id-token: write
  contents: read
```